lib/src/package_analysis/summary.dart

```diff
@@ -0,0 +1,215 @@
+import 'package:analyzer/dart/analysis/results.dart';
+import 'package:analyzer/dart/element/element.dart';
+import 'package:collection/collection.dart';
+import 'package:pana/src/package_analysis/shape.dart';
+import 'package:path/path.dart' as path;
+
+import 'common.dart';
+
+Future<PackageShape> summarizePackage(
+  PackageAnalysisContext packageAnalysisContext,
+  String packageLocation,
+) async {
+  final package =
+      PackageShape(libraries: <LibraryShape>[], classes: <ClassShape>[]);
```

Comment:
Just saying trailing comma is also allowed here, but do whatever you want :rofl: 
```suggestion
  final package = PackageShape(
    libraries: <LibraryShape>[],
    classes: <ClassShape>[]
  );
```